homeassistant/components/tessie/config_flow.py

```diff
@@ -54,3 +59,44 @@ async def async_step_user(
             data_schema=TESSIE_SCHEMA,
             errors=errors,
         )
+
+    async def async_step_reauth(self, user_input: Mapping[str, Any]) -> FlowResult:
+        """Handle re-auth."""
+        self._reauth_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
+        )
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: Mapping[str, Any] | None = None
+    ) -> FlowResult:
+        """Get update API Key from the user."""
+        errors: dict[str, str] = {}
+        assert self._reauth_entry
+        if user_input and CONF_ACCESS_TOKEN in user_input:
```

Comment:
given `CONF_ACCESS_TOKEN` is required does this guard need to be there?

Reply:
Removed both instances.